babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+	presets: ['@babel/preset-react'],
```

Comment:
where are the babel plugins for ES features needed for compilation of the project?